### DISCUSSION TITLE
Any chance of pushing to the Jade App?

### DISCUSSION THREAD

**Discussion by datamystic** [JohanDegraeve/xdripswift#354] [General]
Hi there,

Just wondering if it would be possible to push CGM data to Jade - https://www.jadediabetes.com/api/

Thanks!

Simon

**Comment by paulplant** [comment]
Archiving as very old (and we never checked Discussions to see it)

**Reply by datamystic** [reply]
Hi Paul - where should we put this so that you can action it? The Jade App is very much still alive!
